### DISCUSSION TITLE
Summit videos?

### DISCUSSION THREAD

**Discussion by EricZinda** [delph-in/docs#150] [General]
@fcbond , did we figure out a way to share the videos for a short time?  I’m hoping to take notes on mine …

**Comment by fcbond** [comment]
Hi,

For reasons beyond my ken, they all recorded to my computer, not the zoom
cloud (probably something to do with my university's contract with zoom). I
am copying them to a folder on our server compling.upol.cz  and will link
them to the sessions (in about 30min)

I have recordings for:

01-Syntax-Communicating.mp4
01-Theory-and-Applications.mp4
02-LLMs.mp4
02-Tools-and-Systems.mp4
02-Pragmatics-and-Grammar.mp4
03-Grammatical-Theory-and-Engineering.mp4

@Luis Morgado da Costa ***@***.***>  can you put the
ones you recorded there too?

If anyone apart from Luis and I recorded any, please let me know and we
will get them up there.

On Sun, 7 Jul 2024 at 19:18, EricZinda ***@***.***> wrote:

> @fcbond <https://github.com/fcbond> , did we figure out a way to share
> the videos for a short time? I’m hoping to take notes on mine …
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/150>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by emilymbender** [comment]
Thank you, Francis. If possible, can you send the links over the
discussions functionality, rather than posting them? (The idea is for the
recordings to not be generally world-readable, but for DELPH-IN internal
use only.)

Emily

**Comment by fcbond** [comment]
Hi,

I added .htacess, so you have to log in.

username: name of the organization (no hyphen, all lowercase)
password: name of the city (all lowercase)

I think I have some more recordings, not yet converted.  Conversion is very
slow, I will try to do overnight.

On Mon, 8 Jul 2024 at 10:08, emilymbender ***@***.***> wrote:

> Thank you, Francis. If possible, can you send the links over the
> discussions functionality, rather than posting them? (The idea is for the
> recordings to not be generally world-readable, but for DELPH-IN internal
> use only.)
>
> Emily
>
>
> On Mon, Jul 8, 2024 at 9:36 AM Francis Bond ***@***.***>
> wrote:
>
> > Hi,
> >
> > For reasons beyond my ken, they all recorded to my computer, not the
> zoom
> > cloud (probably something to do with my university's contract with
> zoom).
> > I
> > am copying them to a folder on our server compling.upol.cz and will
> link
> > them to the sessions (in about 30min)
> >
> > I have recordings for:
> >
> > 01-Syntax-Communicating.mp4
> > 01-Theory-and-Applications.mp4
> > 02-LLMs.mp4
> > 02-Tools-and-Systems.mp4
> > 02-Pragmatics-and-Grammar.mp4
> > 03-Grammatical-Theory-and-Engineering.mp4
> >
> > @Luis Morgado da Costa ***@***.***> can you put the
> > ones you recorded there too?
> >
> > If anyone apart from Luis and I recorded any, please let me know and we
> > will get them up there.
> >
> > On Sun, 7 Jul 2024 at 19:18, EricZinda ***@***.***> wrote:
> >
> > > @fcbond <https://github.com/fcbond>
> > <
> https://urldefense.com/v3/__https://github.com/fcbond*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcWW9Wq44$>
>
> > , did we figure out a way to share
> > > the videos for a short time? I’m hoping to take notes on mine …
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/orgs/delph-in/discussions/150>
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gclnoVL14$>,
>
> > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE>
>
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcpvPpbks$>
>
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> > Francis Bond <https://fcbond.github.io/>
> > <
> https://urldefense.com/v3/__https://fcbond.github.io/*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcq1-72xo$>
>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*discussioncomment-9984265__;Iw!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcN1mtM44$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6TYYOOH6PHEVTXGJULZLI6O3AVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUGI3DK__;!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcJlfxy5Y$>
>
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/150#discussioncomment-9984518>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRU6Y2Z47IWS7WC6QF3ZLJCGHAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUGUYTQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by emilymbender** [comment]
Thank you!

**Comment by fcbond** [comment]
Also, as we discussed, I will take them down at the end of September.

On Mon, 8 Jul 2024 at 10:38, emilymbender ***@***.***> wrote:

> Thank you!
>
> On Mon, Jul 8, 2024 at 10:33 AM Francis Bond ***@***.***>
> wrote:
>
> > Hi,
> >
> > I added .htacess, so you have to log in.
> >
> > username: name of the organization (no hyphen, all lowercase)
> > password: name of the city (all lowercase)
> >
> > I think I have some more recordings, not yet converted. Conversion is
> very
> > slow, I will try to do overnight.
> >
> > On Mon, 8 Jul 2024 at 10:08, emilymbender ***@***.***> wrote:
> >
> > > Thank you, Francis. If possible, can you send the links over the
> > > discussions functionality, rather than posting them? (The idea is for
> > the
> > > recordings to not be generally world-readable, but for DELPH-IN
> internal
> > > use only.)
> > >
> > > Emily
> > >
> > >
> > > On Mon, Jul 8, 2024 at 9:36 AM Francis Bond ***@***.***>
> > > wrote:
> > >
> > > > Hi,
> > > >
> > > > For reasons beyond my ken, they all recorded to my computer, not the
> > > zoom
> > > > cloud (probably something to do with my university's contract with
> > > zoom).
> > > > I
> > > > am copying them to a folder on our server compling.upol.cz and will
> > > link
> > > > them to the sessions (in about 30min)
> > > >
> > > > I have recordings for:
> > > >
> > > > 01-Syntax-Communicating.mp4
> > > > 01-Theory-and-Applications.mp4
> > > > 02-LLMs.mp4
> > > > 02-Tools-and-Systems.mp4
> > > > 02-Pragmatics-and-Grammar.mp4
> > > > 03-Grammatical-Theory-and-Engineering.mp4
> > > >
> > > > @Luis Morgado da Costa ***@***.***> can you put the
> > > > ones you recorded there too?
> > > >
> > > > If anyone apart from Luis and I recorded any, please let me know and
> > we
> > > > will get them up there.
> > > >
> > > > On Sun, 7 Jul 2024 at 19:18, EricZinda ***@***.***> wrote:
> > > >
> > > > > @fcbond <https://github.com/fcbond>
> > <
> https://urldefense.com/v3/__https://github.com/fcbond*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8jl9S3Cg$>
>
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://github.com/fcbond*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcWW9Wq44$%3E
> > >
> > > > , did we figure out a way to share
> > > > > the videos for a short time? I’m hoping to take notes on mine …
> > > > >
> > > > > —
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/orgs/delph-in/discussions/150>
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8bOL8-_4$>
>
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gclnoVL14$%3E,
>
> >
> > >
> > > > or unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE>
>
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8SiXJMck$>
>
> > >
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAIPZRVA5BYEJ56CGLDZK7LZLFZ6HAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHEYDOMZYGE*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcpvPpbks$%3E
> > >
> > > > > .
> > > > > You are receiving this because you were mentioned.Message ID:
> > > > > ***@***.***>
> > > > >
> > > >
> > > >
> > > > --
> > > > Francis Bond <https://fcbond.github.io/>
> > <
> https://urldefense.com/v3/__https://fcbond.github.io/*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8unIZ0bs$>
>
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://fcbond.github.io/*3E__;JQ!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcq1-72xo$%3E
> > >
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*discussioncomment-9984265__;Iw!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcN1mtM44$%3E,
>
> >
> > >
> > > > or unsubscribe
> > > > <
> > >
> >
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6TYYOOH6PHEVTXGJULZLI6O3AVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUGI3DK__;!!K-Hz7m0Vt54!iNFwpI4DOQ8DvkhXhhG9ylIqUFfvmvrwbqT11WNDf3uXiLor7zTbryMD33JA3PxEVEuf15FebCrw52gcJlfxy5Y$%3E
> > >
> > > > .
> > > > You are receiving this because you are subscribed to this
> > thread.Message
> > > > ID: ***@***.***>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/orgs/delph-in/discussions/150#discussioncomment-9984518>
>
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*discussioncomment-9984518*3E__;IyU!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8PN2hA24$>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAIPZRU6Y2Z47IWS7WC6QF3ZLJCGHAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUGUYTQ>
>
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AAIPZRU6Y2Z47IWS7WC6QF3ZLJCGHAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUGUYTQ*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8NFokI5U$>
>
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> > Francis Bond <https://fcbond.github.io/>
> > <
> https://urldefense.com/v3/__https://fcbond.github.io/*3E__;JQ!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8unIZ0bs$>
>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/150*discussioncomment-9984774__;Iw!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8mUKic5c$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6QSAH2JC2AV47ERHULZLJFFLAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUG43TI__;!!K-Hz7m0Vt54!iegYd5LgdakvQTGRpPcX6MvbOGBpJR7xmtIxBIf4V0DO3Zn-gUkWfoPsnTUWdjO6XpO4WN4tZkG4P1y8-rIPA1M$>
>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/150#discussioncomment-9984821>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRXXVNFE3Z4ZQCRLK63ZLJFXHAVCNFSM6AAAAABKPQWFTSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOBUHAZDC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by emilymbender** [comment]
Appreciated!
